### DISCUSSION TITLE
App deployment & collaboration

### DISCUSSION THREAD

**Discussion by aclark02-arcus** [Gilead-BioStats/gsm.app#116] [General]
Greetings,

Great app! I'm working on developing a pharma specific R/Shiny repo filled with apps like this one - sort of an "pharma app catalog" if you will. However, before doing so, I'm making sure all applications are deployable. I noticed that this pkg certainly is, but I wanted to ask which branch I should focus on? `main`, `dev`, or something else?

Also, are you open to outside collaborators joining in? I'll reach out if I have any other questions. Thanks!

**Comment by jwildfire** [comment]
Hi @aclark02-arcus - Sounds like a great project. We'd certainly be happy to be included. I'd deploy the `main` branch. That branch gets updated with each release, and *should* be mostly bug-free. `dev` will have any new functionality since the most recent release and won't have been as thoroughly tested. 

Note that the default deploy via `gsmApp::run_app()` for v1.0 will be fairly data-heavy, though we've got some ideas about how to improve that in the long run. Just let us know if you have issues deploying to Shinyapps.io and we can discuss how to work around.

We're definitely open to collaborators - we'll be working on this and other associated RBQM projects through this [Phuse RBQM project](https://advance.phuse.global/display/WEL/OpenRBQM%3A+Pre-Competitive+Collaboration+on+Open-Source+Software+for+RBQM). A new technical work stream is ramping up soon, so feel free to reach out via the contact info in the link if you or others would like to be involved.
